pkg/controller/v1beta1/inferenceservice/reconcilers/cabundleconfigmap/cabundle_configmap_reconciler.go

```diff
@@ -91,10 +90,9 @@ func (c *CaBundleConfigMapReconciler) getCabundleConfigMapForUserNS(caBundleName
 
 	// Check if cabundle configmap exist & the cabundle.crt exist in the data in controller namespace
 	// If it does not exist, return error
-	caBundleConfigMap := &corev1.ConfigMap{}
-	if err := c.client.Get(context.TODO(),
-		types.NamespacedName{Name: caBundleNameInConfig, Namespace: kserveNamespace}, caBundleConfigMap); err == nil {
+	caBundleConfigMap, err := c.clientset.CoreV1().ConfigMaps(kserveNamespace).Get(context.TODO(), caBundleNameInConfig, metav1.GetOptions{})
 
+	if err == nil {
```

Comment:
The else block's error message might be misleading. It should be "cannot get the configmap from the cluster"

Reply:
Done